script.bluetooth.delay/addon.py

```diff
@@ -99,24 +79,32 @@ def speakers():
 def main():
 
     if arg == "reset":
-        xbmc.executebuiltin('PlayerControl(stop)')
-        Addon.setSettingBool('state', 1)
-        reset1()
+        if (xbmc.getCondVisibility('Player.HasVideo') == False):
+            xbmc.executebuiltin("PlayMedia("+path+"reset.mp4,1)")
+        xbmc.executebuiltin('ActivateWindow(busydialognocancel)')
+        time.sleep(2)
+        for x in range (800):
+            xbmc.executebuiltin("Action(AudioDelayPlus)")
+        for x in range (400):
+            xbmc.executebuiltin("Action(AudioDelayMinus)")
+        time.sleep(1)
         skin1()
-        xbmcgui.Dialog().notification("",Addon.getLocalizedString(30006), "",t*2)
+        time.sleep(1)
+        xbmc.executebuiltin('Dialog.Close(busydialognocancel)')
+        Addon.setSettingBool('state', 1)
+        Addon.setSettingBool('reset', 1)
 
     elif d2 == d1:
         if firstRun == "false":
             xbmcgui.Dialog().ok(Addon.getLocalizedString(30007), Addon.getLocalizedString(30008))
             Addon.setSettingBool('firstRun', 1)
         xbmcaddon.Addon().openSettings()
 
-    elif (xbmc.getCondVisibility('Player.HasMedia') == False):
-        Addon.setSettingBool('state', 1)
-        reset1()
-        skin1()
-        xbmcgui.Dialog().notification("",line1, "",t)
-        xbmcgui.Dialog().notification("",Addon.getLocalizedString(30009), "",t)
+    elif (xbmc.getCondVisibility('Player.HasVideo') == False):
+        if state == "true":
+            xbmcgui.Dialog().notification("",line1 + " " + Addon.getLocalizedString(30009), "",t*3)       
```

Comment:
You should use proper python formatting. https://docs.python.org/3/tutorial/inputoutput.html#the-string-format-method

Reply:
Please explain to me what exactly to change?

Reply:
Google python3 fstrings for example, or str format()